slang.h

```diff
@@ -2652,6 +2653,12 @@ namespace slang
                 index);
         }
 
+        VariableReflection* getBindingRangeVariable(SlangInt index)
```

Comment:
Should this be `getBindingRangeLeafVariable`, to match `getBindingRangeLeafTypeLayout`?

Reply:
Done.